### PR TITLE
Removing usage of reproject in RGB documentation

### PR DESCRIPTION
**Pull request by bsipocz**
This PR partly addresses #5570 (the first 3 points, writing a tutorial is clearly beyond the scope of this PR).

Needs https://github.com/astropy/astropy-data/pull/24 to be merged first.

### CODE REVIEW

**Comment by bsipocz** [comment]
@pllim - could you please add milestone 1.3.1, too?

**Comment by bsipocz** [comment]
@larrybradley or @astrofrog - Could you have a look at this?

File: docs/visualization/lupton_rgb.rst [hunk 1]
```diff
@@ -30,59 +30,44 @@ scale and size. Changing ``minimum`` will change the black level, while
 ``stretch`` and ``Q`` will change how the values between black and white are
 scaled.
 
-For a more in-depth example, download the `g`_, `r`_, `i`_ SDSS frames of the
-area around the Hickson 88 group and try the example below (requires the
-`reproject package`_, installable via the astropy conda channel, or via pip),
-and compare it with Figure 1 of Lupton et al. (2004):
+For a more in-depth example, download the ``g``, ``r``, ``i`` SDSS frames
+(the will serve as the blue, green and red channels respectively) of
```
Comment (larrybradley) [inline docs/visualization/lupton_rgb.rst]: "the" -> "they"

**Comment by bsipocz** [comment]
rendered version: http://people.bolyai.elte.hu/~sic/astropy/visualization/lupton_rgb.html

**Comment by pllim** [comment]
I'm probably nitpicking here -- The image from this doc has larger FOV and different orientation than Lupton et al. 2004 Fig. 1, and does not match the SDSS link at all (i.e., I can't tell which one is the galaxy from your example in the latter). From the viewpoint of someone who is assessing whether this is the function I want to use or not, I would like to see:

1. Same FOV and orientation as Lupton et al. 2004 Fig. 1.
2. Given the SDSS UI is limited, no need to have same FOV, but at least center the galaxy of interest.

But you can ignore if package maintainers disagree.

**Comment by bsipocz** [comment]
@pllim - Sounds sensible, however my only aim within this PR was to remove the usage of reproject and get the same result as it was before in the astropy docs: http://docs.astropy.org/en/latest/visualization/lupton_rgb.html

Double checking it I realize that the orientation is upside down, I'm not sure whether that is an issue with the implementation or the with the code that produces the figure in the docs.

**Comment by larrybradley** [comment]
@bsipocz Please update the SDSS link to something in the field, e.g.:
http://skyserver.sdss.org/dr13/en/tools/chart/navi.aspx?ra=313.12381&dec=-5.74611

The link currently in master is wrong.

**Comment by larrybradley** [comment]
@bsipocz The image is flipped because of a bug in `matplotlib 2.0.0` where `imsave` ignores the `origin` keyword (https://github.com/matplotlib/matplotlib/issues/7656).

**Comment by larrybradley** [comment]
@bsipocz  My last comment refers to the jpg output.  To fix the flip in the docs simply add `origin='lower'` to the `imshow` call.

**Comment by bsipocz** [comment]
@larrybradley - Done

**Comment by larrybradley** [comment]
Thanks, @bsipocz!

**Comment by pllim** [comment]
http://docs.astropy.org/en/latest/visualization/lupton_rgb.html :tada:

**Pull request merged**